src/main/java/seedu/address/MainApp.java

```diff
@@ -80,27 +85,47 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         /*Optional<ReadOnlyAddressBook> addressBookOptional;
         ReadOnlyAddressBook initialData;*/
         Optional<ReadOnlyAppointmentBook> appointmentBookOptional;
-        ReadOnlyAppointmentBook initialData;
+        Optional<ReadOnlyPropertyBook> propertyBookOptional;
+        ReadOnlyPropertyBook initialPropertyData;
+        ReadOnlyAppointmentBook initialAppointmentData;
         try {
             //addressBookOptional = storage.readAddressBook();
             appointmentBookOptional = storage.readAppointmentBook();
             //if (!addressBookOptional.isPresent()) {
             if (!appointmentBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample AddressBook");
+                logger.info("Data file not found. Will be starting with a sample AppointmentBook");
             }
             //initialData = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
-            initialData = appointmentBookOptional.orElseGet(SampleDataUtil::getSampleAppointmentBook);
+            initialAppointmentData = appointmentBookOptional.orElseGet(SampleDataUtil::getSampleAppointmentBook);
         } catch (DataConversionException e) {
-            logger.warning("Data file not in the correct format. Will be starting with an empty AddressBook");
+            logger.warning("Data file not in the correct format. Will be starting with an empty Appointment");
```

Comment:
Should `empty Appointment` be `empty AppointmentBook`?

Reply:
done